src/cluster_slot_stats.c

```diff
@@ -157,10 +160,11 @@ static void clusterSlotStatsUpdateNetworkBytesOutForReplication(long long len) {
     len *= (long long)listLength(server.slaves);
     serverAssert(c->slot >= 0 && c->slot < CLUSTER_SLOTS);
     serverAssert(clusterNodeIsMaster(getMyClusterNode()));
+    kvstoreDictMetadata *meta = (kvstoreDictMetadata *)kvstoreEnsureDictMetadata(server.db->keys, c->slot);
     /* We sometimes want to adjust the counter downwards (for example when we want to undo accounting for
      * SELECT commands that don't belong to any slot) so let's make sure we don't underflow the counter. */
-    serverAssert(len >= 0 || server.cluster_slot_stats[c->slot].network_bytes_out >= (uint64_t)-len);
-    server.cluster_slot_stats[c->slot].network_bytes_out += len;
+    serverAssert(len >= 0 || meta->network_bytes_out >= (uint64_t)-len);
```

Comment:
I think we can make it debug assert. there is no real complex logic beyond it that might lead to negative len or negative network_bytes_out